test/BaselineOutput/Common/OVA/OVA-CV-iris-out.txt

```diff
@@ -1,55 +1,49 @@
-maml.exe CV tr=OVA{p=AvgPer{ lr=0.8 }} threads=- norm=No dout=%Output% data=%Data% seed=1 xf=Term{col=Label}
+maml.exe CV tr=OVA{p=AvgPer{ lr=0.8 } useprob=-} threads=- norm=No dout=%Output% data=%Data% seed=1 xf=Term{col=Label}
 Not adding a normalizer.
 Training learner 0
-Training calibrator.
 Training learner 1
-Training calibrator.
 Training learner 2
-Training calibrator.
 Not training a calibrator because it is not needed.
 Not adding a normalizer.
 Training learner 0
-Training calibrator.
 Training learner 1
-Training calibrator.
 Training learner 2
-Training calibrator.
 Not training a calibrator because it is not needed.
 
 Confusion table
           ||========================
 PREDICTED ||     0 |     1 |     2 | Recall
 TRUTH     ||========================
         0 ||    21 |     0 |     0 | 1.0000
-        1 ||     0 |    20 |    10 | 0.6667
+        1 ||     3 |     0 |    27 | 0.0000
         2 ||     0 |     0 |    28 | 1.0000
           ||========================
-Precision ||1.0000 |1.0000 |0.7368 |
-Accuracy(micro-avg): 0.873418
-Accuracy(macro-avg): 0.888889
-Log-loss:           0.393949
-Log-loss reduction: 0.637753
+Precision ||0.8750 |0.0000 |0.5091 |
+Accuracy(micro-avg): 0.620253
```

Comment:
This drop is large. I suppose that this change doesn't dramatically affect its prediction quality... Any idea?

Reply:
Before it was taking an uncalibrated trainer, calibrating it automatically, and then using the calibrated trainer to get these results. With the changes in this PR, it no longer automatically calibrates a trainer. So these results are using the raw values instead of calibrated ones. That is why the drop happens. If you passed in a calibrated trainer, the output would be the same.

Reply:
I know calibrator no longer presents. As a post-processing step, a calibrator should not affect the relative rank of scores. Say, we have three classes, A, B, and C. If a calibrator is appended, their scores might be 0.1, 0.5, 0.8. Without a calibrator, the scores might become -100, 100, 10000. If my assumption is correct, there should be no accuracy change. Did I miss something?

Reply:
I believe this behavior is expected, but I will look more into this and see what I find.

Reply:
So after doing more research, this change matches what is currently in the master branch.
The current file in master is running with a calibrated learner. To confirm my changes are not impacting the accuracy, checked out master, changed the tests that create these to not use probabilities, and ran it. The output matches from master without probabilities matches exactly with this file here.

To repro this, run the `MulticlassReductionTest` in the `TestPredictors` class. Make sure that you update `TestLearners.Ova` and `TestLearners.OvaWithFastForest` to not use probabilities, like so:
`SubComponent("OVA", "p=AvgPer{ lr=0.8 } useProb=-")` and `SubComponent("OVA", "p=FastForest{ } useProb=-")`. These are found in the `Learners.cs` class. If these numbers are not what they should be, then there is a bug in the non calibrated version of these trainers, but if it is a bug it exists in master already and my code is not changing it.

Reply:
So after doing more research, this change matches what is currently in the master branch.
The current file in master is running with a calibrated learner. To confirm my changes are not impacting the accuracy, checked out master, changed the tests that create these to not use probabilities, and ran it. The output matches from master without probabilities matches exactly with this file here.

To repro this, run the `MulticlassReductionTest` in the `TestPredictors` class. Make sure that you update `TestLearners.Ova` and `TestLearners.OvaWithFastForest` to not use probabilities, like so:
`SubComponent("OVA", "p=AvgPer{ lr=0.8 } useProb=-")` and `SubComponent("OVA", "p=FastForest{ } useProb=-")`. These are found in the Learners.cs class. If these numbers are not what they should be, then there is a bug in the non calibrated version of these trainers, but if it is a bug it exists in master already and my code is not changing it.